src/olympia/scanners/admin.py

```diff
@@ -116,6 +116,37 @@ def field_choices(self, field, request, model_admin):
         ]
 
 
+class ExcludeMatchedRuleFilter(SimpleListFilter):
```

Comment:
I would have expected you to inherit from `ScannerRuleListFilter` which takes care of the choices/lookups for you, you just have to customize the queryset. Did you look into that ? Does that not work ?

Reply:
I wasn't able to customize the default entry. With this patch, it looks like this (and that will, hopefully, avoid confusions):

<img width="266" alt="Screen Shot 2020-03-24 at 17 16 43" src="https://user-images.githubusercontent.com/217628/77450221-45938e00-6df3-11ea-95a3-6b2631f46d13.png">

Reply:
Ok